magento/pwa-studio
# [Doc] Update contribution guide

## Description

This PR updates the contribution guide with revisions based on the current workflow.

## Related Issue
Closes PWA-122

## Acceptance 

@tjwiebell 
@awilcoxa 
@Jordaneisenburger (optional)

### Verification Stakeholders

@tjwiebell 
@awilcoxa 
@Jordaneisenburger (optional)

### Specification



### Verification Steps

1. Open `.github/CONTRIBUTING.md`
1. Verify content is accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
